OpenRA.Mods.Common/Widgets/WidgetUtils.cs

```diff
@@ -270,6 +271,24 @@ public static void TruncateButtonToTooltip(ButtonWidget button, string text)
 			else
 				button.GetTooltipText = null;
 		}
+
+		public static void AddSuffixToPlayerNameLabel(LabelWidget label, Player p)
```

Comment:
This method name implies that it's adding the suffix to existing label content. IMO something like `BindPlayerNameAndStatus` would be clearer.